modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -169,8 +330,58 @@ calcOpticalFlowPyrLK(const GArray<GMat>    &prevPyr,
                            int              flags        = 0,
                            double           minEigThresh = 1e-4);
 
+/** @brief Gaussian Mixture-based or K-nearest neighbours-based Background/Foreground Segmentation Algorithm.
+The operation generates a foreground mask.
+
+@return Output image is foreground mask, i.e. 8-bit unsigned 1-channel (binary) matrix @ref CV_8UC1.
+
+@note Functional textual ID is "org.opencv.video.BackgroundSubtractor"
+
+@param src input image: Floating point frame is used without scaling and should be in range [0,255].
+@param bsParams Set of initialization parameters for Background Subtractor kernel.
+*/
+GAPI_EXPORTS GMat BackgroundSubtractor(const GMat& src, const cv::gapi::video::BackgroundSubtractorParams& bsParams);
+
+/** @brief Standard Kalman filter <http://en.wikipedia.org/wiki/Kalman_filter>.
+However, you can modify transitionMatrix, controlMatrix,
+and measurementMatrix to get an extended Kalman filter functionality.
+
+Output image is predicted or corrected state, i.e. 32-bit or 64-bit float 1-channel
+matrix @ref CV_32F or CV_64F.
```

Comment:
It is not true. Depth must be CV_32F or CV_64F exactly.

Reply:
`C1` refers to one channel, and you mention that matrix should be 1-channel float

Reply:
Could you please study https://github.com/opencv/opencv/blob/ef0eed8d3c63b64bacc755dc4ffe5358a2cf30a7/modules/video/include/opencv2/video/tracking.hpp#L368 to clarify this point.

Reply:
Firstly, the main point of this comment was to use `@ref` twice, at every reference,  or otherwise doxygen doesn't configure it as a reference. You can see it in your built doxygen.

Then, in `opencv/modules/video/include/opencv2/video/tracking.hpp` it was probably decided not to mention the number of channels, but as `CV_<SMTH> == CV_<SMTH>C1`, they implicitly mean 1-channel type. OK, why not.
But here it is said `i.e. 32-bit or 64-bit float 1-channel matrix` so in this case I'd prefer to mention `@ref CV_32FC1` and `@ref CV_64FC1` explicitly.

Reply:
The depth is `CV_32F`, but **the type** is `CV_32FC1`

Reply:
In our case CV_32F or CV_64F aren't depth. In OCV implementation they are mentioned as required types.

Reply:
I saw the documentation of them, thank you.
They also didn't mention `32-bit or 64-bit float 1-channel matrix` explicitly.
As I explained, `CV_32FC1` and ` CV_32F` are the same numbers inside, so when they say `type is CV_32F` they mean `CV_32FC1` actually.
All I wanted to say that the precise reference for the phrase `32-bit 1-channel floating point matrix` is `CV_32FC1`.
But I don't actually think it is a necessary change, so let it be as you decided.

The main purpose of this thread was to add the second `@ref`.